packages/codemirror-blocks/src/components/Node.tsx

```diff
@@ -240,6 +240,15 @@ const Node = ({ expandable = true, ...props }: Props) => {
     }
   }, [props.node.id]);
 
+  // If this is the currently focused node, then focus the node element
+  // as soon as we finish rendering.
+  const focusedNode = useSelector(selectors.getFocusedNode);
+  useEffect(() => {
+    if (nodeElementRef.current && focusedNode?.id === props.node.id) {
+      nodeElementRef.current?.focus();
+    }
+  }, [focusedNode?.id, props.node.id]);
+
```

Comment:
My understanding of line 250 is that this hook "subscribes" to the `focusedNode` selector and the node `id`. Does this mean that any time the `focusedNode` changes, _every_ node asks if the focused node's id is the same as the node's id? If not, what does it mean?

Reply:
yep, that's what this means.